packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -161,13 +141,6 @@ const RoundVideoUploadList = ({ election }: { election: CurrentElection }) => {
             </Text>
```

Comment:
suggestion: 

```jsx
<Container>
    <Text>
        It appears you haven't participated in any election round.
        Nothing to upload.
    </Text>
</Container>
```

This will fix the padding issues.